docs/using.md

```diff
@@ -238,13 +249,18 @@ $ minishift start --http-proxy http://YOURPROXY:PORT --https-proxy https://YOURP
                    --https-proxy https://<proxy_username>:<proxy_password>YOURPROXY:PORT
 ```
 
-There is also `--no-proxy` which allows you to specify a comma-separated list of hosts which
+You can also use `--no-proxy` to specify a comma-separated list of hosts which
 should not be proxied. For a list of all available options refer to the
 [synopsis](./minishift_start.md) of the `start` command.
 
 Using the proxy options will transparently configure the Docker daemon as well as OpenShift to
 use the specified proxies.
 
+It will also set the `http-proxy`, `https-proxy` and `no-proxy`
```

Comment:
Careful here. You resolved the merge conflict the wrong way. You brought this sentence back, even though it was removed on master. I'll take care of this.